ui/views.py

```diff
@@ -136,6 +137,16 @@ def __call__(self, request, repo_slug):
         if repo_slug not in set([x.slug for x in repos]):
             return HttpResponseForbidden("unauthorized")
         self.repo = [x for x in repos if x.slug == repo_slug][0]
+        # get sorting from params if it's there
+        params = dict(request.GET.copy())
+        if ('sortby' in params and
```

Comment:
@noisecapella I initially agreed that removing dict was a good idea, but when I do it, my code does not work properly.
I guess the `type(request.GET)` even if it resembles a `dict`, is slightly different. So I will leave it in this way